pmm/pmm2-api-tests.groovy

```diff
@@ -85,16 +85,17 @@ pipeline {
         stage('Run API Test') {
             steps {
                 sh '''
-                    sudo docker run -e PMM_SERVER_URL=\${PMM_URL} --name ${BUILD_TAG} pmm-api-tests
+                    sudo docker run -e PMM_SERVER_URL=\${PMM_URL} -e PMM_RUN_UPDATE_TEST=1 --name ${BUILD_TAG} pmm-api-tests
                 '''
             }
         }
     }
     post {
         always {
             sh '''
+                sudo docker ps -a
                 sudo docker cp ${BUILD_TAG}:/go/src/github.com/Percona-Lab/pmm-api-tests/pmm-api-tests-junit-report.xml ./pmm-api-tests-junit-report.xml
-                ls
+                ls -al
```

Comment:
Was this needed for debugging? Or is intentionally left out?

Reply:
It is both needed for debugging and intentionally left out :) I have an idea why `*.xml` was required, that should help to debug that problem – but only after merging.